src/sentry/integrations/slack/event_endpoint.py

```diff
@@ -165,53 +163,50 @@ def on_message(self, request, integration, token, data):
         return self.respond()
 
     def on_link_shared(self, request, integration, token, data):
-        with sentry_sdk.start_transaction(
-            op="slack.link_shared", name="SlackLinkShared", sampled=1.0
-        ) as span:
-            parsed_issues = defaultdict(dict)
-            event_id_by_url = {}
-            for item in data["links"]:
-                try:
-                    span.set_tag("link", parse_link(item["url"]))
```

Comment:
Perhaps remove the `parse_link` method? We can always add it back.

Reply:
@manuzope I figured not using it would be pretty temporary - we'll either use `parse_link` in Sentry or Kibana right? I can take it out but it'll just go back in soon enough.

Reply:
Sounds good. Okay let's keep it.